### ISSUE TITLE
Dependencies installation failed

### ISSUE THREAD

**Issue by bronzehedwick** [azadkuh/cmus-osx#21]
I tried building on macOS 10.14.2 and Python 3.7.2. When running `pip3 install -r requirements.txt` I get this error:

```
    Modules/_AppKit_inlines.m:9:3: error: "Install Xcode 8.1"
    # error "Install Xcode 8.1"
      ^
    1 error generated.
    error: command 'clang' failed with exit status 1

    ----------------------------------------
Command "/usr/local/opt/python/bin/python3.7 -u -c "import setuptools, tokenize;__file__='/private/var/folders/39/
pgr2dqt5657bd0fny8ydpsm00000gn/T/pip-install-5x41j4mu/pyobjc-framework-Cocoa/setup.py';f=getattr(tokenize, 'open',
 open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --re
cord /private/var/folders/39/pgr2dqt5657bd0fny8ydpsm00000gn/T/pip-record-3aym7t0v/install-record.txt --single-vers
ion-externally-managed --compile" failed with error code 1 in /private/var/folders/39/pgr2dqt5657bd0fny8ydpsm00000
gn/T/pip-install-5x41j4mu/pyobjc-framework-Cocoa/
```

My XCode CLI tools are installed, and is at version 10.1.

Any help would be greatly appreciated.

Thanks!

**Comment by PhilipTrauner** [comment]
Should be resolved as of commit 36d9341, please pull and try again 🙂

**Comment by bronzehedwick** [comment]
That worked, thanks!
